src/renderer/vt/paint.cpp

```diff
@@ -538,7 +553,8 @@ using namespace Microsoft::Console::Types;
     // Write the actual text string. If we're using a soft font, the character
     // set should have already been selected, so we just need to map our internal
     // representation back to ASCII (handled by the _WriteTerminalDrcs method).
-    if (_usingSoftFont) [[unlikely]]
+    if (_usingSoftFont)
+        [[unlikely]]

```

Comment:
Not that it matters that much, but why was this changed? All previous uses of `[[unlikely]]` have been on the same line as the `if`, so it would be nice if we could remain consistent about this.

Reply:
I think this is my version of `clang-format` being overly aggressive 😓